### PR TITLE
Feature/transactions

### PR DESCRIPTION
**Pull request by reddavis**
- Fetch transactions for all institutions that support it
- Updated transactions schema to hold minimum amount of data required (idea is to have a transactions-metadata table to dump everything else into)
- When sync is performed, transactions are also syncd and saved locally
- Update the iOS app so that it builds

### CODE REVIEW

File: Balance/Shared/Data Model/Sqlite Models/Transaction.swift [hunk 1]
```diff
@@ -53,27 +39,14 @@ final class Transaction {
         
         self.currency = result.string(forColumnIndex: 5)
         self.amount = result.long(forColumnIndex: 6)
-        self.altCurrency = result.string(forColumnIndex: 7)
```
Comment (einsteinx2) [inline Balance/Shared/Data Model/Sqlite Models/Transaction.swift]: Do we not receive an alt currency in any of the APIs like we do for accounts?

**Comment by einsteinx2** [comment]
Finally got this passing, will merge now and we can discuss the Transaction.altCurrency later.

File: Balance/Shared/Data Model/Sqlite Models/Transaction.swift (hunk 1 above)
Comment (reddavis) [reply Balance/Shared/Data Model/Sqlite Models/Transaction.swift]: Coinbase is the only one so I figured it made more sense for that to go in the metadata table?

Comment (einsteinx2) [reply Balance/Shared/Data Model/Sqlite Models/Transaction.swift]: That's fair, I figured it was something like that.

**Pull request merged**